src/bin/bool.rs

```diff
@@ -255,10 +255,10 @@ mod test {
     fn get_params(num_iters: usize) -> SynthParams {
         SynthParams {
             seed: 0,
-            n_samples: 2,
+            n_samples: 0,
```

Comment:
What's with all these changes?

Reply:
these parameters just don't match the defaults for some reason, so I changed them here so that the test is the same configuration as `cargo bool --iters 1/2`

Reply:
Got it.